testsuite/MDAnalysisTests/analysis/test_hole2.py

```diff
@@ -73,6 +73,8 @@ def test_relative(self):
             fixed = check_and_fix_long_filename(abspath)
             assert fixed == self.filename
     
+    @pytest.mark.skipif(os.name == 'nt' and sys.maxsize <= 2**32,
+                        reason="FileNotFoundError on Win 32-bit")
     def test_symlink_dir(self, tmpdir):
```

Comment:
Was this also because of not having symlinking privileges, or is it some weirdness with Windows paths and `os.path.join`?

(Just out of interest; I'm not sure hole2 even supports Windows.)